src/kitconcept/contentcreator/creator.py

```diff
@@ -681,15 +682,21 @@ def modify_siteroot(root_info):
     portal = api.portal.get()
     blocks = root_info["blocks"]
     blocks_layout = root_info["blocks_layout"]
+    plone_version = pkg_resources.get_distribution("Plone").version
```

Comment:
@steffenri I would use Products.CMFPlone rather than Plone. It's possible to have a site that just depends on Products.CMFPlone without all the official addons that Plone includes.